tests/suites/test_suite_pk.data

```diff
@@ -1306,6 +1306,82 @@ PSA import into PSA: ECDSA public to pair (bad)
 depends_on:MBEDTLS_PK_HAVE_ECC_KEYS:MBEDTLS_TEST_PSA_ECC_AT_LEAST_ONE_CURVE:MBEDTLS_PK_CAN_ECDSA_SOME
 pk_import_into_psa_fail:MBEDTLS_PK_ECDSA:FROM_PUBLIC:PSA_KEY_TYPE_ECC_KEY_PAIR(MBEDTLS_TEST_PSA_ECC_ONE_FAMILY):0:MBEDTLS_ERR_PK_TYPE_MISMATCH
 
+PSA import into PSA: transparent -> volatile pair
+pk_import_into_psa_lifetime:0:0:1:1:0
```

Comment:
I think there might be a "wrong polarity" on the 4th parameter of this function. The name of the parameter is `to_public`, while the scripts you used to generate seems to put `to_pair` there:
`for to_pair in [('pair', '1'), ('public', '0')]:`
which is the opposite IMO. Am I wrong?

Reply:
As a general rule, I think test data is more readable when we define constants like `TO_PAIR` and `TO_PUBLIC` than use naked 0 and 1, especially in cases like this when there are multiple boolean / small numerical arguments in a row. (Or we could change the test framework to use named arguments: `pk_import_into_psa_lifetime:...:to_pair=1:...` but obviously that's not happening now.)

Reply:
> ~So only the commit message is wrong. I'll amend it.~

Ah, no: `to_public=1` corresponds to `pair` in the description. So the data does need fixing.